ports/moonlight/port.json

```
@@ -19,7 +19,7 @@
         "image": {},
         "rtr": true,
         "runtime": null,
-        "reqs": [],
+        "reqs": [arkos],
```

Missing double quotes around `"arkos"`:
```suggestion
        "reqs": ["arkos"],
```